components/brave_rewards/resources/page/components/settingsPage.tsx

```diff
@@ -88,8 +88,11 @@ class SettingsPage extends React.Component<Props, State> {
   componentDidUpdate (prevProps: Props, prevState: State) {
     if (
       this.props.rewardsData.enabledMain &&
-      prevProps.rewardsData.initializing &&
-      !this.props.rewardsData.initializing
+      (
```

Comment:
This change fixes an edge case where startRewards is not called if initializing is not transitioning from true to false.